recipe/meta.yaml

```diff
@@ -1,35 +1,40 @@
 {% set name = "google-api-core" %}
-{% set version = "2.2.2" %}
+{% set version = "2.10.1" %}
+{% set sha256 = "e16c15a11789bc5a3457afb2818a3540a03f341e6e710d7f9bbf6cde2ef4a7c8" %}
 
 package:
-  name: {{ name|lower }}-split
+  name: {{ name }}
```

Comment:
There's no reason to use a "false" name at the top; the true name will do. That said, due to the following issue, it is necessary to use an explicit output with the same name:
https://github.com/conda/conda-build/issues/4023